src/app/auth/components/auth/auth.component.html

```diff
@@ -18,6 +18,25 @@
         <p class="text archive-data">
           Haadsma Dairy Truck | The Haadsma Dairy Archive
         </p>
+      </div> -->
+      <div class="legacy-lab-container">
+        <div class="legacy-lab-text">
+          <h1 class="header">Legacy Lab</h1>
+          <h2 class="start-date">Begins March 3rd</h2>
+          <p class="text">
+            At Permanent, we believe in the power of preserving your story. Join
+            Legacy Lab, a free four-week course where you'll create a lasting
+            archive and learn through:
+          </p>
+          <ul class="list">
```

Comment:
In theory we don't need some of these classes, right? There's only one kind of list that will ever show up in `.legacy-lab-container`. I think some of these classes are fine (such as `start-date` which conveys more semantic meaning about what the text is doing) but the `ul` tag is *always* going to represent a list so I feel like representing that in the classname is a bit redundant. Same for `list-item` on the `li` tags.